sanic/blueprint_group.py

```diff
@@ -130,6 +151,15 @@
         :param item: New `Blueprint` object.
         :return: None
         """
+        if self._version and item.version and self._version != item.version:
+            raise APIVersionMismatchException(
+                f"API Version Mismatch. Blueprint {item.name} has version {item.version} "
+                f"while Blueprint Group has {self._version}"
+            )
+        if self._version and not item.version:
+            item.version = self._version
+        if self._strict_slashes is not None:
```

Comment:
This should be the same as the version logic. Only apply if it is not set in the `Blueprint`.